static_test_driver/static_test_driver.ino

```diff
@@ -19,80 +19,74 @@
 #include <avr/pgmspace.h>
 
 // LEDs
-#define THERMO1_LED 23
-#define THERMO2_LED 25
-#define THERMO3_LED 27
-#define FORCE_LED 29
-#define PRESSURE_LED 31
-#define STATE_LED 33
-#define STATUS_LED 35
+#define THERMO1_LED 28
+#define THERMO2_LED 30
+#define THERMO3_LED 32
+#define FORCE_LED 42
+#define PRESSURE_FUEL_LED 44
+#define PRESSURE_OX_LED 50 
+#define INLET_TEMP_LED 46 
+#define OUTLET_TEMP_LED 48 
+#define STATE_LED 38 //TODO: Discard STATE_LED code for current sensors revision due to pin conflict. Pin # was changed to satisfy next revision
+#define STATUS_LED 40 //TODO: Discare STATUS_LED code for current sensors revision due to pin conflict. Pin # was changed to satisfy next revision
 
 // Accelerometer
 Adafruit_MMA8451 mma;
 
-// Thermocouple Setup
-#define MAXDO   24
-#define MAXCLK  22
-
-#define MAXCS1  43
-#define MAXCS2  30
-Adafruit_MAX31855 inlet_thermocouple(MAXCLK, MAXCS1, MAXDO);
-Adafruit_MAX31855 outlet_thermocouple(MAXCLK, MAXCS2, MAXDO);
-
-// Thermocouple and pressure setup for MK_2
-#if CONFIGURATION == MK_2
-#define MAXCS3  32
-Adafruit_MAX31855 chamber_thermocouple(MAXCLK, MAXCS3, MAXDO);
+//Analog Temperature Setup
+#define INLET_TEMP A4
+#define OUTLET_TEMP A5
 
+//Pressure Setup
 #define PRESSURE_CALIBRATION_FACTOR 246.58
 #define PRESSURE_OFFSET 118.33
-#define PRESSURE_PIN A1
-
+#define PRESSURE_FUEL A2
+#define PRESSURE_OX A3 
 #define PRESSURE_NUM_HIST_VALS 10
 
-float pressure_hist_vals[PRESSURE_NUM_HIST_VALS];
-int pressure_val_num = 0;
-bool pressure_zero_ready = false;
-float pressure_zero_val = 0;
+float pressure_hist_vals_1[PRESSURE_NUM_HIST_VALS];
```

Comment:
Since we have multiple pressure sensors, maybe turn these into 1D/2D arrays of size 2?  Would make things a bit cleaner.